src/python/zquantum/qcbm/ansatz_utils.py

```diff
@@ -20,6 +24,29 @@ def get_entangling_layer(
         return get_entangling_layer_all_topology(params, n_qubits, entangling_gate)
     elif topology == "line":
         return get_entangling_layer_line_topology(params, n_qubits, entangling_gate)
+    elif topology == "star":
+        center_qubit = topology_kwargs["center_qubit"]
```

Comment:
I strongly recommend defaulting the `center_qubit` to a value, i.e. 0. The main reason for that is that one might simply `change `line to `star` and expect it to work, but in order for the code to not break, one also needs to define `topology_kwargs` with exactly `"center_qubit"` in it. 
You could for example do: 
` center_qubit = 0`
`if topology_kwargs is not None and "center_qubit" in  topology_kwargs:`
    `center_qubit = topology_kwargs["center_qubit"]`